src/app/js/user/index.js

```diff
@@ -57,18 +57,40 @@ export const getRequest = createSelector(
     getToken,
     getCookie,
     (_, props) => props,
-    (token, cookie, { body, method = 'GET', url }) => ({
-        url,
-        body: JSON.stringify(body),
-        credentials: 'same-origin', //auth with include => error 401 https://developer.mozilla.org/fr/docs/Web/API/Request/credentials
-        headers: {
-            Accept: 'application/json',
-            'Content-Type': 'application/json',
-            Authorization: `Bearer ${token}`,
-            Cookie: cookie,
+    (
+        token,
+        cookie,
+        {
+            body,
+            method = 'GET',
+            url,
+            cred = 'same-origin',
+            cook = true,
+            auth = true,
+            head = {
+                Accept: 'application/json',
+                'Content-Type': 'application/json',
+            },
         },
-        method,
-    }),
+    ) => {
+        if (cook) {
+            head['Cookie'] = cookie;
+        }
+        if (auth) {
+            head['Authorization'] = `Bearer ${token}`;
+        }
+        if (typeof body == 'object') {
+            body = JSON.stringify(body);
+        }
+
+        return {
+            url,
+            body,
+            credentials: cred,
```

Comment:
Why did you rename `credentials` parameter to `cred`, and `headers` to `head` (apart from having same length parameters)?
It would have been neat to return 

```js
{
    url,
    body,
    credentials,
    headers,
    method,
}
```